### ISSUE TITLE
Add config file for server settings

### ISSUE THREAD

**Issue by rpappalax** [jrconlin/CatFacts#1]
Currently the push update server URL is set in the catfacts.db > customers database
There is only 1 entry:

push = https://updates.push.services.mozilla.com/push/gAAAAABV0hv-H_ppnjUlzzoa8_LP6bIs6WSuPASVWxfVe4vYnivxp1d6o3UvH5srny24N1t7_HZSQ2HpxIua9xzx2gbjTKLam-fy4j2Ut9yMziTxCLAe8_SElsDo8CBQ6lreLVRYbcBPo6WIdup5MgKzO-kbzq5MLALa0JVymoRMpmW3adxvsb4=

Would like to be able to configure the server to point either at production or staging server:
autopush
Production: https://updates.push.services.mozilla.com
Staging: https://updates-autopush.stage.mozaws.net

Thanks!
![clevercat](https://cloud.githubusercontent.com/assets/3699494/9311942/8684e3bc-44ce-11e5-91fa-a5a740b054f5.gif)

**Comment by jrconlin** [comment]
I think I misunderstood your question. CatFacts doesn't set the Push server, the device does. What happens is that the client app calls the native navigator.push.register() function which returns an endpoint that points to whatever server the device is pointed at. That push endpoint is then sent by the client to the CatFacts server, which just sends facts to that endpoint.

If your device is already set to use Stage or Dev, then Catfacts should use it. 

In short, CatFacts doesn't care what you use to Push.

![Push](https://cloud.githubusercontent.com/assets/35755/9312430/692bf884-44d1-11e5-9613-7b6f9af43dc0.gif)

**Comment by rpappalax** [comment]
OK, even better.  So if I want to point my client to the stage server (using WebIDE), I assume I can re-set dom.push.ServerURL=wss://push.services.mozilla.com/
to
dom.push.ServerURL=wss://push.stage.mozaws.net/ ?

**Comment by jrconlin** [comment]
Yep. that should do it. 
![catpush](https://cloud.githubusercontent.com/assets/35755/9313347/52dfc276-44d7-11e5-92d4-809324d66d5d.gif)
